modules/api/src/main/java/org/apache/ignite/sql/ColumnMetadata.java

```diff
@@ -51,7 +51,7 @@ public interface ColumnMetadata {
      *
      * @return Value type.
      */
-    SqlColumnType type();
+    ColumnType type();
```

Comment:
does for now column type is exactly SQL column type ? If no - we need to change java doc.

Reply:
agree. Fix for all places, except .Net part